test/document_viewer_web/controllers/auth_controller_test.exs

```diff
@@ -56,23 +54,18 @@ defmodule DocumentViewerWeb.AuthControllerTest do
         |> assign(:ueberauth_failure, %Ueberauth.Failure{
           errors: [%Ueberauth.Failure.Error{message: "failed"}]
         })
-        |> get("/auth/cognito/callback")
+        |> get("/auth/keycloak/callback")
 
       assert redirected_to(conn) == "https://www.mbta.com"
     end
 
     test "logs ueberauth failures", %{conn: conn} do
       log =
         capture_log(fn ->
-          conn
-          |> Plug.Test.init_test_session(%{username: "test@mbta.com"})
-          |> assign(:ueberauth_failure, %Ueberauth.Failure{
-            errors: [%Ueberauth.Failure.Error{message: "failed"}]
-          })
-          |> get("/auth/cognito/callback")
+          conn |> get(~p"/auth/keycloak/callback")
         end)
 
-      assert log =~ "Ueberauth error: failed"
+      assert log =~ "Ueberauth error: Cross-Site Request Forgery attack"
```

Comment:
hahah, I don't like it but I was having trouble with this yesterday anyhow.

Reply:
I think ultimately we are trying to test what happens when we get an error in ueberauth, which this does so I'm fine with it.